mimeparse.py

```diff
@@ -11,6 +9,40 @@ class MimeTypeParseException(ValueError):
     pass
 
 
+def _parseparam(s):
+    while s[:1] == ';':
+        s = s[1:]
+        end = s.find(';')
+        while end > 0 and (s.count('"', 0, end) - s.count('\\"', 0, end)) % 2:
+            end = s.find(';', end + 1)
+        if end < 0:
+            end = len(s)
+        f = s[:end]
+        yield f.strip()
+        s = s[end:]
+
+
+def _parse_header(line):
```

Comment:
let's indicate from where _parse_header and _parseparam are taken.

Something like "Vendored version of cgi.parse_header from python 3.12" or something

Reply:
I've added comments.